src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetPreamble.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Tests
+{
+    public class ASCIIEncodingGetPreamble
+    {
+        [Fact]
+        public void GetPreamble()
+        {
+            Assert.Equal(new byte[0], new ASCIIEncoding().GetPreamble());
```

Comment:
I think I'd find Assert.Empty clearer, but YMMV so don't change this if you find this clearer yourself.